src/features/movies/MoviePage/styled.js

```diff
@@ -2,26 +2,34 @@ import { Link } from "react-router-dom";
 import styled from "styled-components";
 
 export const Wrapper = styled.div`
-  max-width: 1368px;
+  max-width: ${({ theme }) => theme.breakpoints.largest};
   margin: 0 auto;
-  display: flex;
-  flex-direction: column;
   background-color: ${({ theme }) => theme.colors.lightGrey};
+  
   @media (max-width: ${({ theme }) => theme.breakpoints.largest}) {
     padding: 16px;
   }
```

Comment:
Wydaje mi sie ze lepiej zamiast tego max-width: largest zostawic wartosc 1368px a padding dodac poprostu jako bazowa wartosc a nie w @media.

Reply:
Słuszna uwaga, trochę odruchowo dodałem tego propsa. To jeśli nie wyciągniemy tego wrappera, może warto zrobić do niego breakpointa. Puddingu osobiście nie nanosiłem i też nie sprawdzałem, ale dobra uwag. Zmienione. ;D